### ISSUE TITLE
Infeasibility not detected

### ISSUE THREAD

**Issue by nrontsis** [oxfordcontrol/osqp#144]
# The problem
`OSQP` does not detect infeasibility on the following problem. `Gurobi`, on the other hand, detects infeasibility directly from the pre-solve phase.

# Minimal Example
Download [data.jld2](https://github.com/oxfordcontrol/osqp/files/3202121/data.jld2.zip) and run in Julia:
```julia
using SparseArrays
using LinearAlgebra
using JLD2
@load "data.jld2" A c
m, n = size(A)

# We want to solve the following problem
# minimize    1/2x'x - c'x
# subject to  Ax = 1
#              x ≥ 0

# Gurobi
using Gurobi
env = Gurobi.Env()
model = gurobi_model(env, H=SparseMatrixCSC(1.0*I, n, n), f=-c, Aeq=A, beq=ones(m), lb=zeros(n))
optimize(model)

# OSQP
using OSQP
A_ = [A; I]
l = [ones(m); zeros(n)]
u = [ones(m); Inf*ones(n)]
model = OSQP.Model()
OSQP.setup!(model; P=SparseMatrixCSC(1.0*I, n, n), q=-c, A=A_, l=l, u=u)
OSQP.solve!(model)
```
The problem arises in doubly stochastic normalisation of [an undirected weighted graph from the SuiteSparse collection](https://sparse.tamu.edu/DIMACS10/ny2010).

# System Info
```
julia 1.1.0
osqp 0.5.0
gurobi 8.1
```

**Comment by bstellato** [comment]
I have updated your comment to include the required packages to run.  

We should check how the infeasibility conditions behave during the iterations. I can already see that OSQP finds large primal objective and residuals fairly soon.

**Comment by bstellato** [comment]
This is fixed in #147
